client/js/lounge.js

```diff
@@ -184,14 +184,23 @@ $(function() {
 		input.val("");
 		resetInputHeight(input.get(0));
 
-		if (text.indexOf("/collapse") === 0) {
-			$(".chan.active .toggle-preview.opened").click();
-			return;
-		}
-
-		if (text.indexOf("/expand") === 0) {
-			$(".chan.active .toggle-preview:not(.opened)").click();
-			return;
+		if (text.indexOf("/") === 0) {
+			const args = text.substr(1).split(" ");
+			const cmd = args.shift().toLowerCase();
+			if (typeof utils.inputCommands[cmd] === "function") {
+				if (cmd === "join") {
```

Comment:
Just pass the args into the command function, don't do the if check.

Join command can just use `args[0]`.